x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -3176,7 +3196,7 @@ describe('Task Runner', () => {
       status: 'warning',
       errorReason: `maxExecutableActions`,
       logAlert: 4,
-      logAction: 3,
+      logAction: 5,
```

Comment:
Yes, that's correct. If there were N actions configured in a rule with the same connector, then only one `execute-action` event log doc per alert was written instead of N